### PR TITLE
[FIX] mrp: fix kits note

### PR DESCRIPTION
**Pull request by brse-odoo**
This PR is a [FIX] addressing @mathieu-mgm's comment on this past [PR #2720](https://github.com/odoo/documentation/commit/426b66ec7b0111823557c255f28d0b1094c07e46).

The fix is addressed in **lines 22-30**. In addition, I also made small changes to language and RST to abide by the newest documentation guidelines.

### CODE REVIEW

File: content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst [hunk 1]
```diff
@@ -2,34 +2,56 @@
...
-Next, set the :guilabel:`Product Type` to :guilabel:`Storable Product`: this allows for management
-of kit inventory. The :guilabel:`Route` designation under the :guilabel:`Inventory` tab does not
-matter, since Odoo uses the routes of the kit's components for replenishment purposes. All other
-parameters for the kit product may be modified according to preference. Finally, click
-:guilabel:`Save`.
+.. note::
+   Although kits should almost always be set to :guilabel:`Consumable`, companies using
+   **Anglo-Saxon** accounting might need to create kits as a :guilabel:`Storable Product`. This is
+   because when processing invoices for kits, the Cost of Goods Sold (COGS) will be posted in
+   accounting journals.
```
[the remark this answers is not in the document]
Comment (mathieu-mgm) [reply content/applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst:30]: @brse-odoo looks ok to me

**Comment by StraubCreative** [comment]
@robodoo delegate=samueljlieber

**Review by samueljlieber — APPROVED** [review]
Nice work @brse-odoo, only found one small fix– an early line break on line 65. I made this correction and approving now 🙂

**Comment by samueljlieber** [comment]
@robodoo retry please 🙏

**Pull request closed without merging**